debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -308,6 +310,14 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
             .withDescription("The maximum number of records that should be loaded into memory while performing a snapshot")
             .withValidation(Field::isNonNegativeInteger);
 
+    public static final Field SNAPSHOT_MODE_TABLES = Field.create("snapshot.table")
```

Comment:
Can't we use different name like snapshot.data.collection to encompass both tables and MongoDB collections?

Reply:
Ah yes, very good point!